src/translations/ru.json

```
@@ -0,0 +1,39 @@
+{
+  "log_in": "Войти",
+  "log_out": "Выйти",
+  "logged_in_as": "Вы вошли как",
+  "logged_in": "Вы вышли",
+  "logging_in": "Вхожу",
+  "logging_out": "Выхожу",
+  "sign_up": "Зарегистрироваться",
+  "signing_up": "Идет регистрация",
+  "forgot_password": "Забыли пароль?",
+  "recover_password": "Восстановить пароль",
+  "send_recovery_email": "Отправить письмо для восстановления",
+  "sending_recovery_email": "Отправляю письмо для восстановления",
```

Here, I would suggest using the indefinite form as well.

```suggestion
  "sending_recovery_email": "Отправление письма для восстановления",
```